evalml/tests/preprocessing_tests/test_balanced_classification_sampler.py

```diff
@@ -82,15 +85,15 @@ def test_classification_imbalanced_balanced_ratio(num_classes, balanced_ratio):
     indices = bcs.fit_resample(X, y)
     X2 = X.loc[indices]
     y2 = y.loc[indices]
-    if balanced_ratio >= 3:
+    if balanced_ratio <= .3:
         # the classes are considered balanced, do nothing
         pd.testing.assert_frame_equal(X, X2)
         pd.testing.assert_series_equal(y, y2)
     else:
         # remove some samples
-        assert len(X2) == {2: (250 * (balanced_ratio + 1)), 3: (200 * (balanced_ratio + 2))}[num_classes]
+        assert len(X2) == {2: (250 + int(250 / balanced_ratio)), 3: (400 + int(200 / balanced_ratio))}[num_classes]
```

Comment:
What happened to the "+ 1" and "+ 2" here?

Reply:
@dsherry The `+1` and `+2` were used to handle the addition of the minority class(es), while the (250 * balanced_ratio) handled the number of majority class samples that we wanted. I opted to simply add `(minority class + (calculation for majority class))` instead, which allowed me to pull out the `+1` and `+2` from the equation. I felt this would be easier to follow.

Reply:
Understood, so the test setup changed. 👍